app/views/repositories/_delete_record_modal.html.erb

```diff
@@ -2,15 +2,15 @@
   <div class="modal-dialog" role="document">
     <div class="modal-content">
       <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <button title="close delete record modal" type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
         <h4 class="modal-title"><%= t("repositories.modal_delete_record.title") %></h4>
       </div>
       <div class="modal-body">
         <%= t("repositories.modal_delete_record.notice") %>
       </div>
       <div class="modal-footer">
-        <button type="button" class="btn btn-default" data-dismiss="modal"><%= t("general.cancel")%></button>
-        <button type="button" class="btn btn-danger" data-dismiss="modal" onclick="onClickDeleteRecord();">
+        <button title="close delete record modal footer" type="button" class="btn btn-default" data-dismiss="modal"><%= t("general.cancel")%></button>
+        <button title="delete record" type="button" class="btn btn-danger" data-dismiss="modal" onclick="onClickDeleteRecord();">
```

Comment:
add "repository" to the above: example "close delete repository record modal",
so it's more specific